src/stirling/source_connectors/socket_tracer/openssl_trace_bpf_test.cc

```diff
@@ -90,6 +91,19 @@ class OpenSSLTraceTest : public SocketTraceBPFTest</* TClientSideTracing */ fals
     sleep(1);
   }
 
+  void SetUp() override {
+    original_openssl_force_raw_fptrs_ = FLAGS_openssl_force_raw_fptrs;
```

Comment:
Since this value is read during `SocketTraceBPFTest::SetUp()`, we can't use `PL_SET_FOR_SCOPE` as discussed [here](https://github.com/pixie-io/pixie/pull/419#discussion_r840967938).

Reply:
Makes sense

Reply:
On second thought, since BaseOpenSSLTraceTest is the only test class in this test, you don't really need to reset the value on the destructor, because you always explicitly set the value during SetUp(), which is safe.

Reply:
But isn't it possible that any other 'trace bpf' tests that use uprobes could run later and they would have whatever the last value that was set be enabled? I thought that had the potential to pollute future tests, so I erred on the side of caution.

Reply:
There's no risk of that because each test is compiled into its own binary. So tests can't interfere with each other. The only risk would be if a new test class was added to `openssl_trace_bpf_test.cc`.

Anyways, not a big deal either way.

Reply:
ah, I see. Since this is an easy fix, I've pushed another commit and retested to remove the `TearDown` method. Thanks for explaining that!